common/version.go

```diff
@@ -4,15 +4,15 @@ import "github.com/nspcc-dev/neo-go/pkg/interop/native/std"
 
 const (
 	major = 0
-	minor = 13
-	patch = 2
+	minor = 14
+	patch = 0
 
 	// Versions from which an update should be performed.
 	// These should be used in a group (so prevMinor can be equal to minor if there are
 	// any migration routines.
 	prevMajor = 0
-	prevMinor = 12
-	prevPatch = 2
+	prevMinor = 13
+	prevPatch = 0
```

Comment:
should it be `0.13.0`?

Reply:
Yes, we are upgrading from `v0.13.1` in N3 Testnet and `v0.13.2` in N3 Mainnet. Those versions are bigger than `v0.13.0`, so it should be okay.